dbt/models/tax/schema.yml

```diff
@@ -23,7 +23,53 @@ sources:
 
       - name: pin
         description: '{{ doc("table_pin") }}'
-
+        columns:  
+          - name: tax_code_num
+            description: '{{ doc("shared_column_tax_code")}}'
+            tests:
+              - row_values_match_after_join:
+                  name: pin_tax_dist_matches_iasworld_legdat
+                  external_model: source('iasworld', 'legdat')
+                  external_column_name: taxdist
+                  column_alias: tax_code_num
+                  external_column_alias: taxdist
+                  group_by: 
+                    - year
+                    - pin
+                  join_condition:
+                    ON  model.pin = external_model.parid
+                    AND model.year = external_model.taxyr
+                    AND external_model.cur = 'Y'
+                    AND external_model.deactivat IS NULL
+                  config:
+                    where:
+                      CAST(year AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
+          - name: class
+            description: '{{ doc("shared_column_char_class")}}'
+            tests:
+              - res_class_matches_pardat:
+                  name: tax_pin_class_equals_iasworld_pardat
+                  parid_column_name: pin
+                  taxyr_column_name: year
+                  additional_select_columns:
+                    - agg_func: max
+                      column: year
+                  config:
+                    where:
+                      CAST(year AS int) BETWEEN {{ var('test_qc_year_start') }} AND {{ var('test_qc_year_end') }}
```

Comment:
@Removing these clauses increases the number of errors by a significant amount. @jeancochrane recommended leaving them in, because that's our operating practice, but it's up to you @dfsnow on if we want to test the historical data.

Reply:
We can keep the clauses in for now.